src/main/java/org/folio/bulkops/domain/bean/Instance.java

```diff
@@ -284,4 +284,9 @@ public String getIdentifier(IdentifierType identifierType) {
   public Integer _version() {
     return version;
   }
+
+  @Override
+  public boolean isMarcInstance() {
+    return source.equals("MARC");
```

Comment:
1) We have constant for source MARC: https://github.com/folio-org/mod-bulk-operations/blob/d35032577575936aceea581c1cea27f7afa31cea/src/main/java/org/folio/bulkops/util/Constants.java#L85 2) I'm not sure if source can be null, but it will be safer from first look to replace MARC.equals(source).